Allow user to specify time of day

wim-beck opened the pull request:
The current version does not support specifying a time of day.
I changed the _possibleChars function to allow this.
I added the ':' and space characters to accomplish this:
Case '@':
n+="0..9 :"
I also formatted the javascript to be able to edit it.

wim-beck committed:
Allow user to specify time

The current version does not support specifying a time of day.
I changed the _possibleChars function to allow this.
I added the ':' and space characters to accomplish this:
Case '@':
n+="0..9 :"

wim-beck committed:
Format javascript

wim-beck committed:
WSP build

pieceofsummer commented:
Editing a jQuery UI library is a very bad idea. Any update will break all your changes.

Adding a plugin like http://trentrichardson.com/examples/timepicker/ is much better, as input validation takes place and you don't need to modify standard libraries.

wim-beck committed:
Allow user to specify time (2)

I reverted the changes to the jquery library and used the plugin you
suggested. Works like a charm. Thanks for the quick feedback and pointer
to the plugin!

wim-beck committed:
Update README.md

wim-beck committed:
Undo javascript formatting

ryannewington committed:
Adds loader for datapicker control and jquery

ryannewington committed:
Updates datepicker control to support MIM 2016 SP1 (and continues to support MIMM 2016 RTM and FIM 2010 R2)
Adds support for conditionally loading the jQuery library
Updates jQuery version to v1.11.3
Updates jQueryUI DatePicker to v1.11.4

ryannewington committed:
Updates WSP

ryannewington committed:
Update README.md

ryannewington committed:
Update README.md

wim-beck committed:
Merge pull request #1 from ryannewington/master

Updates to support MIM 2016 SP1

wim-beck commented:
Updates to support MIM 2016 SP1

ryannewington committed:
Update README.md

wim-beck committed:
Merge pull request #2 from ryannewington/master

Update README.md
